redfish-core/lib/assembly.hpp

```diff
@@ -49,7 +49,6 @@ inline void
 
         // Set the default Status
         tempyArray.at(assemblyIndex)["Status"]["Health"] = "OK";
-        tempyArray.at(assemblyIndex)["Status"]["State"] = "Enabled";
```

Comment:
You could still have this default state then you could get rid of the else on lines 276 -280. I feel that fits more with the existing Status Health and what we do upstream. I am okay here though

Reply:
I will revert this.